spec/lib/healthchecks/queue_latency_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'sidekiq/testing'
+require 'okcomputer'
+require 'healthchecks'
+require 'scholarsphere/redis_config'
+
+RSpec.describe HealthChecks::QueueLatencyCheck do
+  before do
+    Sidekiq::Testing.disable!
+    redis_config = Scholarsphere::RedisConfig.new
+    Sidekiq.configure_server do |config|
+      config.redis = redis_config.to_hash
+    end
+
+    Sidekiq.configure_client do |config|
+      config.redis = redis_config.to_hash
+    end
+  end
+
+  describe '#check' do
+    context 'when latency is fine' do
```

Comment:
What about when the latency is not "fine" ?

Reply:
I'm not sure how to mock out an item on the queue, and fake it's date. I'd be happy to add that path though.

Reply:
We could look at this today during refactoring, if you'd like.